homeassistant/components/squeezebox/coordinator.py

```diff
@@ -70,3 +74,46 @@ def _prepare_status_data(self, data: dict) -> dict:
 
         _LOGGER.debug("Processed serverstatus %s=%s", self.lms.name, data)
         return data
+
+
+class SqueezeBoxPlayerUpdateCoordinator(DataUpdateCoordinator):
+    """Coordinator for Squeezebox players."""
+
+    def __init__(self, hass: HomeAssistant, player: Player, server_uuid: str) -> None:
+        """Initialize the coordinator."""
+        super().__init__(
+            hass,
+            _LOGGER,
+            name=player.name,
+            update_interval=timedelta(seconds=PLAYER_UPDATE_INTERVAL),
+            always_update=True,
+        )
+        self.player = player
+        self.available = True
+        self._remove_dispatcher: Callable | None = None
+        self.server_uuid = server_uuid
+
+    async def _async_update_data(self) -> dict:
```

Comment:
Can we type the dict better?

Reply:
Yes, right now we don't put anything in it but in the next PR (alarms support) we put a key "alarms" in it with an Alarms dict as the response.

I think dict[str, Any] is probably appropriate.